nodeup/pkg/model/network.go

```diff
@@ -1,123 +0,0 @@
-/*
-Copyright 2019 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package model
-
-import (
-	"fmt"
-	"path/filepath"
-
-	"k8s.io/klog"
-	"k8s.io/kops/pkg/systemd"
-	"k8s.io/kops/upup/pkg/fi"
-	"k8s.io/kops/upup/pkg/fi/nodeup/nodetasks"
-)
-
-// NetworkBuilder writes CNI assets
-type NetworkBuilder struct {
-	*NodeupModelContext
-}
-
-var _ fi.ModelBuilder = &NetworkBuilder{}
-
-// Build is responsible for configuring the network cni
-func (b *NetworkBuilder) Build(c *fi.ModelBuilderContext) error {
-	var assetNames []string
-
-	// @TODO need to clean up this code, it isn't the easiest to read
-	networking := b.Cluster.Spec.Networking
-	if networking == nil || networking.Classic != nil {
-	} else if networking.Kubenet != nil || networking.GCE != nil {
```

Comment:
Where did the asset adding for GCE go? Is `KubenetBuilder` supposed to also act for `networking.GCE`?

Reply:
Thanks. I didn't realise that GCE was kubenet.

Reply:
This is replaced with a generic check for kubenet-based providers now.

Reply:
Comment resolved by subsequent commit.